desktop/menus/file-menu.js

```diff
@@ -5,38 +5,42 @@ const { appCommandSender } = require('./utils');
 const buildFileMenu = (isAuthenticated) => {
   isAuthenticated = isAuthenticated || false;
 
-  const submenu = [
-    {
-      label: '&New Note',
-      visible: isAuthenticated,
-      accelerator: 'CommandOrControl+Shift+I',
-      click: appCommandSender({ action: 'newNote' }),
-    },
-    ...(isAuthenticated ? [{ type: 'separator' }] : []),
-    {
-      label: '&Import Notes…',
-      visible: isAuthenticated,
-      click: appCommandSender({
-        action: 'showDialog',
-        dialog: 'IMPORT',
-      }),
-    },
-    {
-      label: '&Export Notes…',
-      visible: isAuthenticated,
-      accelerator: 'CommandOrControl+Shift+E',
-      click: appCommandSender({
-        action: 'exportNotes',
-      }),
-    },
-    ...(isAuthenticated ? [{ type: 'separator' }] : []),
-    {
-      label: '&Print…',
-      visible: isAuthenticated,
-      accelerator: 'CommandOrControl+P',
-      click: appCommandSender({ action: 'printNote' }),
-    },
-  ];
+  let submenu = [];
+
+  if (isAuthenticated) {
+    submenu = [
+      {
+        label: '&New Note',
+        visible: isAuthenticated,
+        accelerator: 'CommandOrControl+Shift+I',
+        click: appCommandSender({ action: 'newNote' }),
+      },
+      ...(isAuthenticated ? [{ type: 'separator' }] : []),
+      {
+        label: '&Import Notes…',
+        visible: isAuthenticated,
+        click: appCommandSender({
+          action: 'showDialog',
+          dialog: 'IMPORT',
+        }),
+      },
+      {
+        label: '&Export Notes…',
+        visible: isAuthenticated,
+        accelerator: 'CommandOrControl+Shift+E',
+        click: appCommandSender({
+          action: 'exportNotes',
+        }),
+      },
+      ...(isAuthenticated ? [{ type: 'separator' }] : []),
+      {
+        label: '&Print…',
+        visible: isAuthenticated,
```

Comment:
Since we are inside an if statement based on `isAuthenticated` we should be able to remove all the `isAuthenticated` checks from inside. 
This would be applicable to the edit menu code as well.

Reply:
Thank you for the review.

Fix in the lastest commit.

Reply:
Thanks again. It still looks like inside the `isAuthenticated` if blocks we could change all these lines `...(isAuthenticated ? [{ type: 'separator' }] : []),` to be just `{ type: 'separator' }` instead.

Reply:
fix in the last commit